### ISSUE TITLE
Handle unreadable data objects

### ISSUE THREAD

**Issue by clnsmth** [IMCR-Hackathon/datapie#36]
Not all data objects can be read by `data_package_read()`. Perhaps a default value should be assigned to the data objects that can't be read (e.g. "Sorry, data_object_name.ext is an unsupported data type and cannot be read at this time.") so the UI can inform the user of this issue. What do you think @wetlandscapes, @ktoddbrown, @sheilasaia, @lkuiucsb ?

**Comment by clnsmth** [comment]
What do you think @sheilasaia and @wetlandscapes?

**Comment by sheilasaia** [comment]
Maybe we can discuss as a group at the next meeting whether people think this is alpha or beta, @clnsmth? I'm more inclined to say alpha but could be persuaded if we're trying to get out the package asap/have a soft-release of the package.

**Comment by clnsmth** [comment]
IMO this is a basic feature that should be implemented before the public RFC (beta-release) to preempt the question every user will be asking of us (i.e. "Why isn't `datapie` allowing me to view X data object."). 

One approach is to offer all data object names to the user for selection in the GUI drop down list, and when an unsupported object is selected, an informative message is returned to the user. We could implement this by having `data_package_read()` output the object list as is but updated to include the names of unsupported objects each accompanied by a message string. The GUI read/parse logic would have to be updated to pass this string to the user as a message and fail gracefully.

**Comment by clnsmth** [comment]
Data types that cannot be read by `data_package_read()` are now removed from the output list and corresponding notifications are printed to the "Raw Data" tab of the UI.

**Comment by clnsmth** [comment]
So much for my "fix". I've commented out this section of `data_package_read()` and am working on a more robust solution now.

**Comment by clnsmth** [comment]
Take 2.

**Comment by atn38** [comment]
@clnsmth, `doi:10.6073/pasta/ea5c14e500ab121b254ee99f47a8affb` cannot be read into the GUI because parsing failed for `.data` type files. Is this issue still relevant?

**Comment by clnsmth** [comment]
Yes, this issue is still relevant @atn38. But rather than trying to address this on our end, it may be simpler to address this in `metajam`.

**Comment by atn38** [comment]
working on this in branch `fix_36`

**Comment by clnsmth** [comment]
https://github.com/IMCR-Hackathon/datapie/issues/72#issuecomment-520888432 is more closely related to the work being done here, and will therefore not be resolved by fix_72.
